src/idom/_console/rewrite_key_declarations.py

```diff
@@ -22,7 +21,7 @@
 
 @click.command()
 @click.argument("paths", nargs=-1, type=click.Path(exists=True))
-def update_html_usages(paths: list[str]) -> None:
+def rewrite_key_declarations(paths: list[str]) -> None:
```

Comment:
maybe just `rewrite_keys` instead?

But we might also want this to handle the underscore syntax for keys, so perhaps this becomes `v1_syntax_update`